lua/telescope/builtin/internal.lua

```diff
@@ -951,15 +951,18 @@ internal.jumplist = function(opts)
   -- reverse the list
   local sorted_jumplist = {}
   for i = #jumplist, 1, -1 do
-    jumplist[i].text = ''
-    table.insert(sorted_jumplist, jumplist[i])
+    if vim.api.nvim_buf_is_valid(jumplist[i].bufnr) then
+      jumplist[i].text = vim.api.nvim_buf_get_lines(jumplist[i].bufnr, jumplist[i].lnum, jumplist[i].lnum+1,
+        false)[1] or ''
```

Comment:
I think this or just doesnt do anything. doesnt matter tho :laughing:

Reply:
It does something in my testing. If I `bdelete` a buffer, than the buffer is still valid, but we can't get the lines for it. Only when I `bwipeout` a buffer does a buffer become invalid. I'm not 100% on the behavior of buffers in vim and neovim, but I think `bdelete` keeps a lot of the buffer data around, but unloads the file text from memory.